manuals/1.0/en/quick-start.md

```
@@ -4,59 +4,81 @@ title: Quick Start
-* In the script editor, select `File` > `Export...` and save the file to the `Application` folder with the file format as application.
-* * Run and select an ALPS file to start the ASD server. Drag & Drop is also supported.
+* In Script Editor, select `File` > `Export...`, choose `Applications` as the location and set file format to `Application`.
+* Run it and select an ALPS file to start the ASD server. Drag & drop is also supported.
+
+## GitHub Actions Version
+
+Create ASD in CI. For details, please see [https://github.com/marketplace/actions/app-state-diagram](https://github.com/marketplace/actions/app-state-diagram).
+
+## Execution Options
+
+```
```

Markdownlint

Fenced code blocks should have a language specified

(MD040, fenced-code-language)